ogre2/src/Ogre2RenderEngine.cc

```diff
@@ -362,7 +366,7 @@ void Ogre2RenderEngine::CreateLogger()
 
   // create actual log
   this->ogreLogManager = new Ogre::LogManager();
-  this->ogreLogManager->createLog(logPath, true, false, false);
+  this->ogreLogManager->createLog(logPath, true, true, false);
```

Comment:
can we change the debug output back to `false`?

Reply:
Fixed in 25d8719d